hrm-service/.sequelizerc

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
 module.exports = {
-  'config': path.resolve('dist', 'config', 'config.js'),
+  'config': path.resolve('dist', 'src', 'config', 'config.js'),
```

Comment:
I can't start local server without this change, since `npm run build` compiles to this for me.. Let me know if this breaks in other environments.
![Screenshot from 2022-05-18 15-34-00](https://user-images.githubusercontent.com/15805319/169119292-59f1b9dc-0e0a-406e-b6c8-f1add72bcbaa.png)

Reply:
I'm pretty sure that would break it elsewhere...

This is my dist folder:
![hrm-dist](https://user-images.githubusercontent.com/1694716/169122243-3ee34cbe-23cf-41a6-84b7-dc661cd1fd2d.png)

Why is it different for your local machine I wonder? 🤔

Reply:
Check your versions of the twilio cli, and the flex plugin to the cli (not our flex-plugin, the cli plugin). I got an error recently that I couldn't build because my cli plugin version wasn't right.

Reply:
@nickhurlburt  - this is HRM, so there's no Twilio involved. It looks to be something wonky about how I set up the Typescript compilation, but not sure what...

Reply:
@GPaoloni - you aren't running NPM from a different working directory somehow are you? :-P

Reply:
Nono I run `npm run build` from `hrm/hrm-service`, which I think is the right one.

Reply:
Sorry, I don't know why your local version is building like that then? Don't think it's a Linux thing because it seems to build into the correct directory on the deployments

Reply:
Thanks for helping with this, addressed in https://github.com/techmatters/hrm/pull/196/commits/71907411aad14741d61ec071e8cc09235df23db3 :)